src/txn/txn_recover.c

```diff
@@ -1016,6 +1016,13 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
         r.metadata_only = true;
         r.backup_only = false;
     }
+
+    /*
+     * Start metadata recovery. Don't allow any Btree files to be opened, they depend on metadata
+     * that might be modified during recovery.
+     */
+    F_SET_ATOMIC_32(conn, WT_CONN_RECOVERING);
```

Comment:
This is supposed to be WT_CONN_RECOVERING_METADATA, right ?

Reply:
Yes, thanks!  Fixed. in [f07f629](https://github.com/wiredtiger/wiredtiger/pull/11873/commits/f07f629bbf2f72badfa66402ac5770967c26140a)